bc_obps/service/compliance/elicensing/elicensing_api_client.py

```diff
@@ -133,6 +133,76 @@ class InvoiceCreationRequest:
     fees: List[str]
 
 
+@dataclass
```

Comment:
I suggest moving the dataclasses to a separate file to help keep this code more organized.